### DISCUSSION TITLE
Get daily list of changed items!

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#12640] [Archive]
Hi all!

Let's say I want to generate a list every evening of all Product Items where the status field has been changed from "draft" to "published" that day. What would be the easiest way to get this done? Should I use revisions?

Cheers!

Kees.

**Comment by rijkvanzanten** [comment]
Great question! I think checking against activity to find the updates, and then cross referencing the status field in revisions to find the draft->published change is one way. Another approach is to use a `publish_date` field rather than a draft/published status dropdown, that way you don't need to search through anything to find that change 🙂

**Reply by keesvanbemmel** [reply]
Thanks Rijk, date is easiest solution for now!
